### PR TITLE
Write new task: Configuring Pod Initialization.

### PR DESCRIPTION
**Pull request by steveperry-53**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2157)

### CODE REVIEW

**Comment by jaredbhatti** [comment]
Looks good!

File: docs/tasks/configure-pod-container/configure-pod-initialization.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+{% capture steps %}
+
+### Creating a Pod that has an init Container
+
+In this exercise you create a Pod that has one application Container and one
+init Container. The init Container runs to completion before the application
+container starts.
+
+Here is the configuration file for the Pod:
```
Comment (pwittrock) [inline docs/tasks/configure-pod-container/configure-pod-initialization.md]: Should we talk about why this is in an annotation somewhere (e.g. is is an alpha feature)?

File: docs/tasks/configure-pod-container/configure-pod-initialization.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+title: Configuring Pod Initialization
+---
+
+{% capture overview %}
+This page shows how to use an init Container to initialize a Pod before the
+application Container runs.
+
+{% endcapture %}
+
+{% capture prerequisites %}
```
Comment (pwittrock) [inline docs/tasks/configure-pod-container/configure-pod-initialization.md]: Should this have communicating between containers in the same Pod as a prereq?

File: docs/tasks/configure-pod-container/configure-pod-initialization.md (hunk 1 above)
Comment (pwittrock) [inline docs/tasks/configure-pod-container/configure-pod-initialization.md]: We should probably a doc about what happens when features move out of alpha.  We should also talk about how alpha features aren't guaranteed to not-break between releases.

**Comment by steveperry-53** [comment]
> Should we talk about why this is in an annotation somewhere (e.g. is is an alpha feature)?

@pwittrock, Done. I think it's beta.

> Should this have communicating between containers in the same Pod as a prereq?

@pwittrock, The two topics seem like companion topics to me, not one before the other. So I put a link to the other topic under What's Next.

> We should probably a doc about what happens when features move out of alpha. We should also talk about how alpha features aren't guaranteed to not-break between releases.

@pwittrock, @jaredbhatti has been putting some thought into this. Coming soon, I think.

**Pull request merged**